packages/env/lib/build-docker-compose-config.js

```diff
@@ -254,6 +254,7 @@ module.exports = function buildDockerComposeConfig( config ) {
 				image: developmentWpCliImage,
 				volumes: developmentMounts,
 				user: cliUser,
+				command: 'tail -F dne', // Keeps the service alive.
```

Comment:
Why is this needed for the CLI services but not the others?

Reply:
It solves an issue where the cli services just exit immediately after starting them. I think the other services we start (wordpress and tests-wordpress) boot a long-running server, which is probably why they probably stay alive internally. This basically tells the service do nothing indefinitely, until exec does something with it

Reply:
Generally speaking, Docker containers exist to run a single command. Once a container finishes executing its last command it will close. In long-lived services you would exec the daemon, such as apache. You can use commands like this to give the CLI the job of "running forever".